deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -61,8 +73,21 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
         return false;
     }
 
+    private CircuitBreakerConfiguration getCircuitBreakerConfiguration(CodeGenContext context) {
+        UnaryOperator<String> nameToValuePropertyMapper = propertyName -> context.config().getValue(propertyName,
```

Comment:
If you have the static method with config  arguments, you can write
`return CircuitBreakerConfigurationParser.of(context.config());`